### PR TITLE
Add Django 2.1 to supported versions

### LINKED ISSUE #5: Add support for Django 2.1
As far as I can tell, there is no reason for the restriction Django<2.1.

### PR DESCRIPTION
**Pull request by David-Wobrock**
Fixes #5 
I see nothing against supporting Django 2.1

A pity there are no tests at all yet

### CODE REVIEW

**Comment by David-Wobrock** [comment]
It is possible to have just `Django>=1.10`, without upper boundary - but we cannot guarantee that all Django 2 versions will function.
There might be a core change on the migrations/operations in a later Django 2 version which we cannot foresee. And once tests will be implemented, we should gradually add new Django versions I guess

**Comment by igeligel** [comment]
This is exactly why Django switched to semantic versioning. To make sure that minor and patch versions do not break: https://semver.org/ Those versions just introduce new methods to existing public interfaces and bug fixes/improvements to current versions. If we never trust any other software vendor we should build everything ourself...

**Comment by David-Wobrock** [comment]
I understand your point, but this is not about trusting the DSF or not.
In an ideal world, no mistake would happen and no API is broken in a minor version. However, since humans seem to make mistakes from time to time, let's be safe and increase the versions as they come.
The cost of adding a new Django version to the dependencies and to the test matrix here is minimal, and this library will therefore always support what it has been tested against.

That would be my suggestion, since it is also the standard way of handling new versions in other Django libraries.

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
     long_description = f.read()
 
 install_requirements = [
-    'Django>=1.10,<2.1'
+    'Django>=1.10,<2.2'
```
Comment (mes3yd) [inline setup.py]: Let's drop 1.10 while we're at it, as it's no longer a supported DSF release.

File: setup.py [hunk 2]
```diff
@@ -50,6 +50,7 @@
         'Framework :: Django :: 1.10',
```
Comment (mes3yd) [inline setup.py]: See previous.

**Comment by David-Wobrock** [comment]
@mes3yd 
Done :heavy_check_mark: - dropped Django 1.10
Even though I would have kept it while it didn't need any additional effort to maintain it

**Comment by unknownlighter** [comment]
is there any progress for merging this PR and release new version to pypi?

**Comment by David-Wobrock** [comment]
I guess #8 should be preferred, since this PR brings no value to the project

**Comment by mes3yd** [comment]
This has been addressed in different PR.

**Pull request closed without merging**